www/content/tutorials/preview-mode/create-oauth-client.mdx

```diff
@@ -27,9 +27,14 @@ As from `next-drupal 1.5`, user roles are used for OAuth scopes. The scopes are
 
 Next, assign the following permissions to the newly created role.
 
-- Bypass content access control
-- Issue subrequests
-- View user information
+
```

Comment:
The tests run eslint and prettier format checks against the code base. And the tests are failing because you've accidentally added an extra blank line to this mdx file.

Reply:
Oh sorry, I was just editing through the web ui and didn't notice. Hopefully resolved now!